pom.xml

```diff
@@ -10,7 +10,8 @@
 
   <groupId>com.fasterxml.jackson</groupId>
   <artifactId>jackson-parent</artifactId>
-  <version>2.9.1.2-SNAPSHOT</version>
+  <version>2.9.1.2</version>
```

Comment:
Please don't do this: mvn release plugin requires snapshot version, and we'll get 2.9.1.2 release once that occurs.